src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,62 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    public static final int MIN_LOGIN_LENGTH = 6;
+    public static final int MIN_PASSWORD_LENGTH = 6;
+    public static final int MIN_AGE = 18;
+
+    private static final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            return null;
```

Comment:
throw RegistrationException

Reply:
Shouldn't it be a `RuntimeException` instead? I think the absence of an user (`user == null`) is more like an issue with unproper handling a request to register an user. Like, i imagine an user pressing 'Sign Up' button without filling any required filelds - in that case the `User` class object with empty fields should be sent to `register` method, but not null. In that case an user gets an error - 'Login must be present' but not 'User should be present'